sdks/godot/gen.go

```diff
@@ -0,0 +1,150 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+	"text/template"
+)
+
+//go:generate go run gen.go
+//go:generate gdformat addons/com.google.agones/AgonesAlpha.gd
+//go:generate gdformat addons/com.google.agones/AgonesBeta.gd
+//go:generate gdformat addons/com.google.agones/AgonesSdk.gd
+
+const directory = "addons/com.google.agones"
+
+var swaggerFiles = []string{"sdk", "alpha", "beta"}
+
+func main() {
+
+	if _, err := os.Stat(directory); os.IsNotExist(err) {
+		mkErr := os.MkdirAll(directory, os.ModePerm)
+		if !os.IsExist(mkErr) {
+			log.Fatalf("Failed to create directory: %s", mkErr)
+		}
+	}
+
+	for _, swaggerFile := range swaggerFiles {
+
+		log.Println(strings.ToUpper(swaggerFile))
```

Comment:
Is this for debugging only? Or is it useful to leave in the final version?

Reply:
I guess this and the following comment would be relevant to how much data you want the generator to output.

Reply:
Removing for now.